### PR TITLE
Kamille poems

### PR DESCRIPTION
**Pull request by kamamile-tea**

### CODE REVIEW

File: Kamille [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Oh squirrel 
+why do you sit outside my window?
+making me look weird infront of my class
```
Comment (jkschulz) [inline Kamille]: Can you please add a space to "infront"? (--> "in front")

**Review by jkschulz — CHANGES_REQUESTED** [review]
This is great!

File: Kamille (hunk 1 above)
Comment (kamamile-tea) [reply Kamille]: I added the space!

**Review by jkschulz — APPROVED** [review]
Looks great!

**Pull request merged**